runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatChanges.java

```diff
@@ -352,6 +365,19 @@ private void init()
 		add(combo(heal(HITPOINTS, perc(.15, 1)), heal(RUN_ENERGY, 100)), BANDAGES_25202);
 		add(combo(boost(ATTACK, perc(.15, 5)), boost(STRENGTH, perc(.15, 5)), boost(DEFENCE, perc(.15, 5)), boost(RANGED, perc(.15, 5)), boost(MAGIC, perc(.15, 5)), heal(PRAYER, perc(.25, 8))), POTION_OF_POWER1, POTION_OF_POWER2, POTION_OF_POWER3, POTION_OF_POWER4);
 
+		// Moths & Butterflies
+		final Effect rubyHarvest = boost(ATTACK, perc(.15, 4));
+		final Effect snowyKnight = heal(HITPOINTS, perc(.15, 4));
```

Comment:
The wiki seems to have a lot of conflicting information about how much this actually heals. Has this been tested?

Reply:
There was a hotfix that changed it shortly after release of the mixes which changed it which could leave to mixed info. Just tested: Snowy Knight Mix restores 8 and Snowy Knight restores 15. I believe this is a flat amount not percent but my alt doesn't have members currently to test so could only verify on an account with 99 hp

Reply:
@capslock13 you can currently test by changing your stats on beta worlds.